automated install/basic-install.sh

```diff
@@ -543,11 +543,10 @@ installScripts() {
 	${SUDO} cp /etc/.pihole/advanced/Scripts/blacklist.sh /opt/pihole/blacklist.sh
```

Comment:
This all probably should be redone with `install` instead, I think it would flow better.